graphql/server.go

```diff
@@ -45,7 +29,7 @@ func (s *server) registerEchoMutation(schema *schemabuilder.Schema) {
 // registerGame registers the game type.
 func (s *server) registerGame(schema *schemabuilder.Schema) {
```

Comment:
Long term when we actually have stuff here I bet this also breaks into a bunch of files.

Reply:
I agree. Decided not to go crazy yet, but I hope it gets split into more files as it scales.